hack/test-stirunimage.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+function time_now()
+{
+  echo $(date +%s000)
+}
+
+function cleanup()
+{
+    set +e
+    # use sigint so that sti post processing will remove docker container
+    kill -2 "${STI_PID}"
+    echo
+    echo "Complete"
+}
+
+set +e
+img_count=$(docker images | grep -c sti_test/sti-fake)
+set -e
+
+if [ "${img_count}" != "10" ]; then
+    echo "You do not have necessary test images, be sure to run 'hack/build-test-images.sh' beforehand."
+    exit 1
+fi
+
+trap cleanup EXIT SIGINT
+
+echo
+echo 
+echo
+
+sti build git://github.com/bparees/openshift-jee-sample openshift/wildfly-8-centos test-jee-app --run=true &> "${STI_ROOT}/hack/sti-run.log" &
+export STI_PID=$!
+TIME_SEC=1000
+TIME_MIN=$((60 * $TIME_SEC))
+max_wait=2*TIME_MIN
+echo "waiting up to ${max_wait}"
+expire=$(($(time_now) + $max_wait))
+
+set +e
+while [[ $(time_now) -lt $expire ]]; do
+    grep  "as a result of the --run=true option" "${STI_ROOT}/hack/sti-run.log"
+    if [ $? -eq 0 ]; then
+      set -e
+      echo "[INFO] Success running command sti --run=true"
+      rm -f  "${STI_ROOT}/hack/sti-run.log"
+      exit 0
+    fi
+    sleep 1
+done
+
+echo "[INFO] Problem with sti --run=true, see ${STI_ROOT}/hack/sti-run.log"
```

Comment:
yeah, i'd just remove it in the existing cleanup method so it always gets removed.